inspector/hardhat.config.ts

```diff
@@ -73,4 +82,105 @@ const config: HardhatUserConfig = {
   },
 };
 
+task("inspect", "Main task")
+  .addParam("service", "The service to use", "all")
+  .setAction(async (taskArgs, hre: HardhatRuntimeEnvironment) => {
+    const { ethers, deployments, network } = hre;
+    const { service } = taskArgs;
+
+    const ACC_ID =
+      network.name == "baobab"
+        ? (process.env.BAOBAB_ACC_ID as string)
+        : (process.env.CYPRESS_ACC_ID as string);
+    const explorerBaseUrl =
+      network.name == "baobab"
+        ? "https://baobab.klaytnfinder.io/tx"
+        : "https://klaytnfinder.io/tx";
+
+    const _inspectorConsumer = await deployments.get("InspectorConsumer");
+    const inspectorConsumer = await ethers.getContractAt(
+      _inspectorConsumer.abi,
+      _inspectorConsumer.address
+    );
+
+    if (service == "rr") {
```

Comment:
It probably is not worth refactoring, but I would recommend to reuse more code next time. Currently, it looks like there are three branches (`rr`, `vrf` and both). Each of them have separate request pattern, but all of them have the same verification process. It could have been more maintainable if the verification process is reused by all at the end of this function.